images/prow-image/root/usr/bin/deprovision.sh

```diff
@@ -43,6 +43,7 @@ echo "Running the post-installation steps..."
 set +e
 set -x
 bash ./dhcp-pxe-conf/baremetal-lab-post-dhcp-pxe-conf-commands.sh
+PROVISIONING_HOST="${AUX_HOST}" bash ../ipi/deprovision/baremetal-lab-ipi-deprovision-commands.sh
```

Comment:
```suggestion
PROVISIONING_HOST="${AUX_HOST}" PROVISIONING_HOST_SSH_PORT=22 bash ./provisioning-network/baremetal-lab-post-provisioning-network-commands.sh
PROVISIONING_HOST="${AUX_HOST}" PROVISIONING_HOST_SSH_PORT=2222 bash ./provisioning-network/baremetal-lab-post-provisioning-network-commands.sh
```

hey @sgoveas thanks, let's do it this way as for the provisioning network to support a behavior that is going to be developed in a few days from now.